frontend/src/components/worksheets/BundleDetail/BundleDetailSideBar.jsx

```diff
@@ -83,6 +98,42 @@ class BundleDetailSideBar extends React.Component<
             ? renderDuration(bundleInfo.metadata.time)
             : "-- --";
 
+
+        // Collect complete bundle detail info
+        // Sort the metadata by key.
+        let keys = [];
+        let items = [
+            (<div>
+                    <ConfigLabel label="UUID:" inline={true}/>
+                    <div className={classes.dataText}>
+                        {bundleInfo.uuid}
+                    </div>
+                </div>
+            ),
+            (<div>
+                <ConfigLabel label="Is anonymous:" inline={true}/>
+                <div className={classes.dataText}>
+                    {renderFormat(bundleInfo.is_anonymous, 'bool')}
+                </div>
+            </div>)];
+        const repeatedFields = ['data_size', 'description', 'name', 'tags', 'time'];
+        for (let property in metadata) {
+            if (metadata.hasOwnProperty(property) && !repeatedFields.includes(property))
+                keys.push(property);
+        }
+        keys.sort();
+        for (let i = 0; i < keys.length; i++) {
+            let key = keys[i];
+            items.push(
+                <div>
+                    <ConfigLabel label={this.renderInfoLabel(key)} inline={true}/>
+                    <div className={classes.dataText}>
+                        {renderFormat(metadata[key], metadataType[key])}

```

Comment:
Are we sure we want to do this in the first place? I'm not sure we really need to show all metadata fields inline; it might look not good if the metadata field is very large, for example, error_traceback.

Was this a suggestion from an earlier demo?

Reply:
yes, it is a suggestion in last Thursday's dry run demo.